src/api/serializers.py

```diff
@@ -63,6 +64,42 @@ class Meta:
     )
 
 
+class PreprintAuthorSerializer(serializers.HyperlinkedModelSerializer):
+
+    class Meta:
+        model = repository_models.Author
+        fields = ('first_name', 'middle_name', 'last_name', 'affiliation', 'orcid',)
+
+
+class PreprintSubjectSerializer(serializers.HyperlinkedModelSerializer):
+
+    class Meta:
+        model = repository_models.Subject
+        fields = ('name',)
+
```

Comment:
should I have done this with `serializers.ReadOnlyField` rather than defining a `PreprintSubjectSerializer`?